lib/spack/spack/test/conftest.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Copyrght 2013-2019 Lawrence Livermore National Security, LLC and other
```

Comment:
The typo above is what Travis is complaining about

Reply:
Interesting.  Thanks!

Reply:
Fixed.